lib/tools/emacs/erlang.el

```diff
@@ -4,7 +4,7 @@
 ;; Author:   Anders Lindgren
 ;; Keywords: erlang, languages, processes
 ;; Date:     2011-12-11
-;; Version:  2.7.0
+;; Version:  2.8.0-cvs
```

Comment:
Can you set 2.8.0, otherwise this will need to be bumped in the release, which I most certainly will forget.
Also while at it rebase to maint branch for inclusion in OTP-20.1.